code/modules/research/techweb/all_nodes.dm

```diff
@@ -203,14 +203,14 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 3000)
 	export_price = 5000
 
-/datum/techweb_node/basic_meteor_defense
-	id = "basic_meteor_defense"
-	display_name = "Meteor Defense Research"
-	description = "Unlock the potential of the mysterious of why CC decided to not build these around the station themselves."
-	prereq_ids = list("adv_engi", "high_efficiency")
-	design_ids = list("meteor_defence", "meteor_console")
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
-	export_price = 5000
+//datum/techweb_node/basic_meteor_defense
```

Comment:
/* at the start and */ at the end to make a multiline comment

Reply:
I asked kevinz previously how he wanted multiline comments done, and he said it didn't matter. So, looking below, I used the previously commented out part as reference and went with single line comments.